test.sh

```diff
@@ -52,9 +52,7 @@ tc_client()
 	tc -n "${ns}" filter show dev "${iface}" egress
 	tc -n "${ns}" filter show dev "${iface}" ingress
 
-	ip netns exec "${ns}" ethtool -K "${iface}" gro off gso off tso off lro off ufo off sg off
-	# ip -n "${NS}_cli" link set "cpe" gso_max_segs 1 # but perf impact in this particular setup
-	ip netns exec "${NS}_cli" ethtool -K "cpe" gro off gso off tso off lro off ufo off sg off
+	ip -n "${NS}_cli" link set "cpe" gso_max_segs 0
```

Comment:
The perf impact is quite measurable, but that's for a good reason. Maybe we should document that the test env is a bit particular, and in such setup, we don't need to modify the end client/server, but that's OK, that's not the targeted use-case.